cms/static/js/collections/asset.js

```diff
@@ -37,8 +37,7 @@ define([
             return PagingCollection.prototype.parse.call(this, response, options);
         },
 
-        /* jshint unused:false */
-        parseState: function (response, queryParams, state, options) {
+        parseState: function(response, queryParams, state, options) {  // eslint-disable-line no-unused-vars
```

Comment:
My preference here is to remove parameters that are not used. I think that's common enough in JS to be clearer than having to add a lint decorator.